app/views/shared/_rating.html.haml

```diff
@@ -0,0 +1,9 @@
+%h4 Rate This Proposal
+%span Not Recommended
+- (1..5).each do |score|
+  = form_tag '/rate' do
+    = hidden_field_tag :slug, proposal.event.slug
+    = hidden_field_tag :proposal_id, proposal.id
+    = hidden_field_tag :score, score
```

Comment:
Could we make this a drop down so we can show them their saved score? Or indicate which one they selected somewhere in this partial for later reference?